CharlieBackend.Api/Controllers/StudentsController.cs

```diff
@@ -158,5 +155,19 @@ public async Task<ActionResult> DisableStudent(long id)
 
             return disabledStudentModel.ToActionResult();
         }
+
+        /// <summary>
+        /// Gets filtered list of lessons for student
+        /// </summary>
+        /// <response code="200">Returned filtered list of lessons for student </response>
+        [SwaggerResponse(200, type: typeof(IList<LessonDto>))]
+        [Authorize(Roles = "Admin, Mentor, Secretary")]
```

Comment:
Since we are returning a collection of lessons for the current user, shouldn't a user with a "Student" role be able to access this endpoint?
```suggestion
        [Authorize(Roles = "Student")]
```
And why there are roles of admin, mentor and secretary? They won't work here anyway.

Reply:
If you didn't allow `Student` role to access this endpoint, and your implementation of `GetLessonsForStudentAsync()` requires a valid `_currentUserService.AccountId` to find a `Student` entity - it means your endpoint doesn't even work.

You didn't even make sure your endpoint works, did you?

Reply:
Allow `Student` role to access this endpoint

Reply:
Provided additional explanation of a problem here: https://github.com/ita-social-projects/WhatBackend/pull/368#discussion_r582810228